tfhe-zk-pok/src/curve_api/bls12_381.rs

```diff
@@ -36,7 +36,17 @@ fn bigint_to_bytes(x: [u64; 6]) -> [u8; 6 * 8] {
 mod g1 {
     use super::*;
 
-    #[derive(Copy, Clone, PartialEq, Eq, Serialize, Deserialize, Hash)]
+    #[derive(
+        Copy,
+        Clone,
+        PartialEq,
+        Eq,
+        Serialize,
+        Deserialize,
+        Hash,
+        CanonicalSerialize,
+        CanonicalDeserialize,
+    )]
```

Comment:
is the serde attibute still required on the inner ? I'm really not sure how the ark serialization stuff is supposed to work, if you have the info from the doc e.g. don't hesitate to share